pkgs/development/compilers/polyml/default.nix

```diff
@@ -24,12 +31,9 @@ stdenv.mkDerivation rec {
     "--with-gmp"
   ];
 
-  src = fetchFromGitHub {
-    owner = "polyml";
-    repo = "polyml";
-    rev = "v${version}";
-    sha256 = "sha256-72wm8dt+Id59A5058mVE5P9TkXW5/LZRthZoxUustVA=";
-  };
+  doCheck = true;
+
+  checkPhase = "make check";
```

Comment:
You probably don't need to specify this. The default hook already checks for `make check` and `make test`, and runs whichever one it finds.

https://github.com/NixOS/nixpkgs/blob/bf76c9f706c23d8f39991a0c785fa882e0966064/pkgs/stdenv/generic/setup.sh#L1327-L1334

Reply:
Indeed, initially I didn't add the `checkPhase = "make check"` line, but that didn't work for some reason. Instead, I got:
```
running tests
no Makefile or custom checkPhase, doing nothing
```

This is why I added that line.